drivers/exec/driver_test.go

```diff
@@ -160,7 +161,71 @@ func TestExecDriver_StartWaitStop(t *testing.T) {
 		status, err := harness.InspectTask(task.ID)
 		require.NoError(err)
 		require.Equal(drivers.TaskStateExited, status.State)
-	case <-time.After(1 * time.Second):
+	case <-time.After(10 * time.Second):
+		require.Fail("timeout waiting for task to shutdown")
+	}
+
+	require.NoError(harness.DestroyTask(task.ID, true))
```

Comment:
Should `harness.DestroyTask` be called in a defer to run it even on test failure? It's usually best to cleanup even on failures.